test/unit/asrouter/templates/ExtensionDoorhanger.test.jsx

```diff
@@ -0,0 +1,33 @@
+import schema from "content-src/asrouter/templates/CFR/templates/ExtensionDoorhanger.schema.json";
+
+const DEFAULT_CONTENT = {
+  "notification_text": "Recommendation",
+  "heading_text": "Recommended Extension",
+  "info_icon": {
+    "label": "why_seeing_this",
+    "url": "https://support.mozilla.org"
+  },
+  "addon": {
+    "title": "Addon name",
+    "icon": "https://mozilla.org/icon",
+    "author": "Author name"
+  },
+  "text": "Description of addon",
+  "amo_url": "https://example.com",
```

Comment:
move this under the `addon` section, since this is going to come from the API eventually